### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Brandonbr1**
improved CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1 +1,9 @@
+# Contributing to StopModReposts
+
+A big welcome and thank you for considering contributing to StopModReposts open source project! It’s people like you that make it a reality for users in our community.
+
+when CONTRIBUTING you must agree to the following terms terms:<br>
```
Comment (dada513) [inline CONTRIBUTING.md:5]:
```suggestion
When contributing you must agree to the following terms terms:  
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
 By contributing, you grant us full permission to use your work in any way, including, but not limited to: redistribution, relicensing, modifying, etc.
+
+# Code of Conduct
+We at StopModReposts  take our open source community very seriously so please read our [ Code of Conduct](https://github.com/StopModReposts/Illegal-Mod-Sites/blob/master/CODE_OF_CONDUCT.md)
```
Comment (dada513) [inline CONTRIBUTING.md:9]:
```suggestion
# Code of Conduct
We at StopModReposts take our open source community very seriously, so please read our [Code of Conduct](https://github.com/StopModReposts/Illegal-Mod-Sites/blob/master/CODE_OF_CONDUCT.md)
```

**Review by dada513 — CHANGES_REQUESTED** [review]

**Comment by Brandonbr1** [comment]
@dada513  the Changes are done!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1 +1,9 @@
+# Contributing to StopModReposts
+
+A big welcome and thank you for considering contributing to StopModReposts open source project! It’s people like you that make it a reality for users in our community.
```
Comment (dada513) [inline CONTRIBUTING.md:3]:
```suggestion
A big welcome and thank you for considering contributing to the StopModReposts open source project! It’s people like you that make it a reality for users in our community.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1 +1,9 @@
...
+
+When contributing you must agree to the following terms terms:
```
Comment (dada513) [inline CONTRIBUTING.md:5]:
```suggestion
When contributing you must agree to the following terms:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1 +1,9 @@
...
 By contributing, you grant us full permission to use your work in any way, including, but not limited to: redistribution, relicensing, modifying, etc.
+
+# Code of Conduct
+We at StopModReposts take our open source community very seriously, so please read our [Code of Conduct](https://github.com/StopModReposts/Illegal-Mod-Sites/blob/master/CODE_OF_CONDUCT.md)
```
Comment (dada513) [inline CONTRIBUTING.md:9]:
```suggestion
We at StopModReposts take our open source community very seriously, so please read our [Code of Conduct](https://github.com/StopModReposts/Illegal-Mod-Sites/blob/master/CODE_OF_CONDUCT.md) before contributing
```

**Review by dada513 — COMMENTED** [review]
some finishing touches

**Comment by dada513** [comment]
By the way I believe when I do a suggestion in a review you can just apply the patch via a button

**Comment by Brandonbr1** [comment]
> By the way I believe when I do a suggestion in a review you can just apply the patch via a button

@dada513 yea you can do a patch with just a button lol

anyways the changes are done!

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1 +1,9 @@
+# Contributing to StopModReposts
+
+A big welcome and thank you for considering contributing to the StopModReposts open source project! It’s people like you that make it a reality for users in our community.
+
+When contributing you must agree to the following terms:
```
Comment (dada513) [inline CONTRIBUTING.md:5]:
```suggestion
When contributing you must agree to the following terms:  
```
Just noticed this one. By the way to do new lines in markdown you need two spaces

**Comment by dada513** [comment]
@berrysauce I also think we should review the part about you grant us permissions to relicense (which was present before this PR too), as that isn't a healthy permission for an open source project

**Pull request closed without merging**